telemetry/qml/DialogButton.qml

```diff
@@ -40,6 +40,16 @@ Rectangle {
         GradientStop { position: 1.0; color: "#C4C9CD" }
     }
 
+    Keys.onPressed: {
+        if (event.key === Qt.Key_Return ||
+            event.key === Qt.Key_Enter) {
+
+            root.clicked()
+
+            event.accepted = true

```

Comment:
Does that mean "Enter" or "Return" accepts telemetry? If so this is bad, really bad.

Reply:
Well, it needs to be the same as whatever the default button is.  I don't know that it particularly matters which is the default as long as the buttons are clearly labeled for sighted users and screen-reader-friendly for blind ones.  But yeah, we'll get complaints if the default button isn't "no".

Reply:
There is no default button in that dialog. At least there was none before this PR

Reply:
I’m fine with keeping no default, so Enter doesn’t need to work *until* you actively make a selection via the arrow keys.  Then it does need to apply the selected button. 

Still the bigger problem of this dialog being able to be reached at all needs fixing first.